gtfs/spec/en/reference.md

```diff
@@ -176,14 +176,14 @@ File: **Required**
 |  trip_id | **Required** | The **trip_id** field contains an ID that identifies a trip. This value is referenced from the [trips.txt](#tripstxt) file. |  |
-|  arrival_time | **Required** | The **arrival_time** specifies the arrival time at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service date. For times occurring after midnight on the service date, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.  If this stop isn't a time point, use an empty string value for the **arrival_time** and **departure_time** fields or provide an interpolated time, indicated by providing a value of 0 for the **timepoint** field. Stops without arrival times will be scheduled based on the nearest preceding timed stop. To ensure accurate routing, please provide arrival and departure times for all stops that are time points. You must specify arrival and departure times for the first and last stops in a trip.  Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
+|  arrival_time | **Required** | The **arrival_time** specifies the arrival time at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service date. For times occurring after midnight on the service date, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.  If this stop isn't a time point, use an empty string value for the **arrival_time** field or provide an interpolated time, indicated by providing a value of 0 for the **timepoint** field. If interpolated times are indicated with timepoint=0, then times that are strictly adhered to by the transit vehicle (i.e., timepoints) must be indicated with a value of 1 for the timepoint field. To ensure accurate routing, please provide arrival and departure times for all stops that are time points. You must specify arrival and departure times for the first and last stops in a trip.  Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
```

Comment:
@fpurcell , I am going to to incorporate your language with a change, below. I removed the reference to "accurate routing" because GTFS is now used for applications besides trip planning.

@devinbraun , @abyrd : What do you think of this definition of the a time point?

> If this stop is not a timepoint, use either an empty string value for the **arrival_time** field or provide an interpolated time (further, indicate that interpolated times are provided via the **timepoint** field with a value of zero). If interpolated times are indicated with **timepoint**=0, then stops at which the vehicle will hold until the scheduled departure time (i.e., timepoints) must be indicated with a value of 1 for the **timepoint** field. Provide arrival and departure times for all stops that are time points.

Interesting note: GTFS is referenced in the [Wikipedia article "Timing Point".](https://en.wikipedia.org/wiki/Timing_point)